### PR TITLE
feat(shared-data, components): add OT-3 deckmap component

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

This PR:
- Adds actual OT-3 deckmap svg layers to the OT-3 deck def (before they were copied and pasted from the OT-2 deck def lol)
- Updated the `RobotWorkSpace` component to be able to deserialize JSON-ified svg layers from `svgson`
- Updated the OT-2 deck def to make use of JSON serialized SVG via `svgson` (this is how our module defs get rendered)
- Updated the deck schema (and TS types) to leverage this new layering interface (no geometry changed at all, just presentational) 

closes RCORE-250

# Changelog

- Add OT-3 deckmap componoent
- Update deck schema layering key

# Review requests

- check storybook and make sure all deck defs actually render. we should probably make separate OT-2 and OT-3 stories cuz their layer props are different.
- go through the app and make sure OT-2 deckmaps still render as expected 

# Risk assessment

Med

### CODE REVIEW

File: components/src/hardware-sim/Deck/RobotWorkSpace.stories.tsx [hunk 1]
```diff
@@ -19,7 +29,10 @@ export default {
       defaultValue: 'ot2_standard',
     },
     deckLayerBlocklist: {
-      options: Object.keys(allDeckDefs.ot2_standard.layers),
+      options: [
+        ...getLayerIds(allDeckDefs.ot2_standard.layers),
+        ...getLayerIds(allDeckDefs.ot3_standard.layers),
```
Comment (shlokamin) [inline components/src/hardware-sim/Deck/RobotWorkSpace.stories.tsx:34]: we should probs make this story into 2 different ones for OT-2 and OT-3, cuz having options show up that are not relevant for both is kinda weird

**Review by koji — COMMENTED** [review]
Some parts of the OT-3 deck are cut off. Is this an expected behavior?

Screen Shot 2022-10-10 at 11 56 09 AM

**Comment by brenthagen** [comment]
> Some parts of the OT-3 deck are cut off. Is this an expected behavior?
> 
> Screen Shot 2022-10-10 at 11 56 09 AM

looks like this is an issue with the dimensions/cornerOffsetFromOrigin in the ot3_standard definition

File: components/src/hardware-sim/Deck/RobotWorkSpace.stories.tsx [hunk 2]
```diff
@@ -3,11 +3,21 @@ import { RobotWorkSpace } from './RobotWorkSpace'
 import { RobotCoordsForeignDiv, Module } from '@opentrons/components'
 import { getModuleDef2 } from '@opentrons/shared-data'
 import { getDeckDefinitions } from './getDeckDefinitions'
+import type { DeckLayer } from '@opentrons/shared-data'
```
Comment (brenthagen) [inline components/src/hardware-sim/Deck/RobotWorkSpace.stories.tsx]: this should be type INode from svgson

Comment (shlokamin) [reply components/src/hardware-sim/Deck/RobotWorkSpace.stories.tsx]: oh ya you're right... weird that TS checks didnt catch this

**Comment by shlokamin** [comment]
@koji the viewport is taken from the deck def dimensions which are wrong...i am going to push an update to the deck def as a follow up (it'll need hardware approval)

**Review by koji — APPROVED** [review]
Looks good to me.

**Pull request merged**